main.go

```diff
@@ -73,6 +76,7 @@ func getRepos(dir, user, maxrepo string, limit []string) (repos []string) {
 
 		// All if 'limit' slice empty or if 'repo' exists in 'limit' slice
 		if !(len(limit) == 0 || inSlise(repo, limit)) {
+			log.Printf("skip %s", repo)
```

Comment:
Remove this line.
If you have more than 20-30 repositories but want backup one than you'll see 20-30 messages... 
So remove this line:  log.Printf("skip %s", repo)